addon/data/postmessageProxyFrame.js

```diff
@@ -0,0 +1,17 @@
+const TELEMETRY_QUEUE_POLL_INTERVAL = 5000;
+
+// Periodically poll for queued telemetry pings
+setInterval(function() {
+  document.getElementById('tp-proxy').contentWindow.postMessage({
+    op: 'fetchTelemetryPings'
+  }, '*');
+}, TELEMETRY_QUEUE_POLL_INTERVAL);
+
+// Listen for postMessage
+window.addEventListener('message', function(ev) {
```

Comment:
And would we want to check for `ev.origin == ?` here? (Though I'm not sure what the origin will be when the message comes from the add-on and/or web extension.)